wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/settings-edit.php

```diff
@@ -96,13 +116,28 @@
 	<tr>
 		<th><label for="default_sort[deepl_api_key]">
 				<?php esc_html_e( 'Deepl Free API Key', 'glotpress' ); ?>
+			</label>
+					<?php
+					if ( gp_array_get( $gp_external_translations, 'deepl_translations_used', 0 ) > 0 ) {
+						echo '<br>';
+						echo '<small>';
+						/* translators: %d: number of OpenAI translations used. */
+						echo esc_html( sprintf( _n( '%d Deepl translation used', '%d Deepl translations used', 'glotpress' ), gp_array_get( $gp_external_translations, 'deepl_translations_used', 0 ) ) );
+						if ( gp_array_get( $gp_external_translations, 'deepl_same_translations_used', 0 ) > 0 ) {
+							/* translators: %d: number of OpenAI translations used. */
+							echo ' ' . esc_html( sprintf( _n( '(%d used without modifications)', '(%d used without modifications)', 'glotpress' ), gp_array_get( $gp_external_translations, 'deepl_same_translations_used', 0 ) ) );
+						}
+						echo '</small>';
+					}
+					?>
+				<br>
 				<a href="https://www.deepl.com/account/usage" target="_blank"><small>
 						<?php
 						/* translators: %d: number of chars translated with Deepl. */
 						echo esc_html( sprintf( __( '(Chars translated with Deepl: %d)', 'glotpress' ), gp_array_get( $gp_default_sort, 'deepl_chars_used', 0 ) ) );
```

Comment:
These parenthesis should also be removed for consistency.

Reply:
Done!